components/codec/src/error.rs

```diff
@@ -11,6 +11,9 @@ pub enum ErrorInner {
 
     #[fail(display = "Data padding is incorrect")]
     BadPadding,
+
+    #[fail(display = "bad format key(length)")]
+    KeyLength,
```

Comment:
Prefer not to introduce this variant, since it is not used anywhere in the codec itself.

Reply:
done